core/types/transaction_marshalling.go

```diff
@@ -164,8 +164,8 @@ func (tx *Transaction) MarshalJSON() ([]byte, error) {
 		enc.IsSystemTx = &itx.IsSystemTransaction
 		// other fields will show up as null.
 
+	// [Kroma: START]
 	case *KromaDepositTx:
```

Comment:
How about this? `depositTxWithNonce` also
```suggestion
	case *KromaDepositTx, *kromaDepositTxWithNonce:
```